MSAL/test/unit/utils/MSALTestURLSession.m

```diff
@@ -371,6 +371,11 @@ - (NSURLSessionDataTask *)dataTaskWithRequest:(NSURLRequest *)request
 
 
 #pragma mark - NSURLSession
+
+// Invalidate
+- (void)invalidateAndCancel{}
```

Comment:
nit: either add a space after "Cancel" or put the curly braces on their own lines